src/main/java/com/uber/jenkins/phabricator/PhabricatorNotifier.java

```diff
@@ -231,6 +236,23 @@ public final boolean perform(final AbstractBuild<?, ?> build, final Launcher lau
         return true;
     }
 
+    protected UberallsClient getUberallsClient(Logger logger, String gitUrl, String branch) {
+        if (uberallsClient != null) {
+            return uberallsClient;
+        }
+        return new UberallsClient(
+                getDescriptor().getUberallsURL(),
+                logger,
+                gitUrl,
+                branch
+        );
+    }
+
+    // Just for testing
+    protected void setUberallsClient(UberallsClient client) {
```

Comment:
the other thing I could do is set up a test http server to return mocks from this, but that seemed kinda heavy -- any other ideas?